src/pages/index.js

```diff
@@ -95,3 +173,33 @@ addButton.addEventListener("click", function () {
   formValidators["card-form"].resetValidation();
   cardForm.open();
 });
+
+avatarButton.addEventListener("click", function () {
+  formValidators["avatar-form"].resetValidation();
+  avatarForm.open();
+});
+
+/* ************************* */
+/* Promises for initial data */
+/* ************************* */
+const initialCards = api
+  .getCardInfo()
+  .then((res) => (res.ok ? res.json() : res.status))
+  .then((res) => {
+    cardGallery.setItems(res);
+    cardGallery.renderItems();
+  })
+  .catch((err) => console.error(err));
+
+const initialProfile = api
+  .getUserInfo()
+  .then((res) => (res.ok ? res.json() : res.status))
+  .then((res) => {
+    userInfo.setUserInfo({ name: res.name, about: res.about });
+    userInfo.setAvatar(res.avatar);
+  })
+  .catch((err) => console.error(err));
+
+Promise.all([initialCards, initialProfile])
+  .then((res) => (res.ok ? res.json() : res.status))
+  .catch((err) => console.error(err));
```

Comment:
You don't need this code. It has no effect


But if you want to use it then it should be done correctly

More info is here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

```js

Promise.all([api.getUserInfo(), api.getCardInfo()])
// destructure the response
  .then(([userData, cards]) => {
      // set all the data
  })
  .catch(err => {
    // catch possible errors
  });

```